package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gus-markdown",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Then just bump the package.json up to match. The version number of the released extension should match the version number of the codebase.